docs/dyndns/update-api.rst

```diff
@@ -185,17 +187,31 @@ Basic authentication with forced use of IPv4 (will remove IPv6 address from the
   curl --ipv4 https://update.dedyn.io/?hostname=<your domain> \
     --header "Authorization: Token <your token secret>"
 
-Basic authentication with forced use of IPv6 (will remove IPv4 address from the DNS)::
+Basic authentication with forced use of IPv4 (while preserving the IPv6 address from the DNS)::
 
-  curl --ipv6 https://update.dedyn.io/?hostname=<your domain> \
+  curl --ipv4 "https://update.dedyn.io/?hostname=<your domain>&myipv6=preserve" \
     --header "Authorization: Token <your token secret>"
 
+Basic authentication with forced use of IPv6 (will remove IPv4 address from the DNS), option #1::
+
   curl --user <your domain>:<your token secret> https://update6.dedyn.io/
 
-Basic authentication with simultaneous update of IPv4 and IPv6::
+or option #2::
+
+  curl --ipv6 https://update.dedyn.io/?hostname=<your domain> \
+    --header "Authorization: Token <your token secret>"
+
+Basic authentication with forced use of IPv6 (while preserving the IPv4 address from the DNS)::
+
+  curl --ipv6 "https://update.dedyn.io/?hostname=<your domain>&myipv4=preserve" \
+    --header "Authorization: Token <your token secret>"
```

Comment:
Using `update6.dedyn.io` is preferred over `--ipv6` (because the latter is a curl flag, and the former works with all clients).